package.json

```diff
@@ -9,8 +9,7 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "node node_modules/jest/bin/jest.js",
-    "test:watch": "node node_modules/jest/bin/jest.js --watch",
-    "postinstall": "react-native upgrade; react-native link"
```

Comment:
You are sure we don't need that ? At least the react-native link is linking the native dependencies unless I miss something ?

Reply:
I added that line. However I don't think we need to run these commands on every install. React-native link should be to link native dependencies. We might need to run it when we install new libraries. But we shouldn't have to do it after every npm i

Reply:
Indeed you're correct the linking is just done in the os directories config files.